lib/dry/configurable/nested_config.rb

```diff
@@ -0,0 +1,31 @@
+module Dry
+  module Configurable
+    # @private
+    class NestedConfig
+      def initialize(klass)
+        @klass = klass
+      end
+
+        # @private no, really...
+      def create_config
+        if @klass.instance_variables.include?(:@_config)
+          @klass.__send__(:create_config)
+        end
+      end
+
+      private
+
+      def config
+        @klass.config
+      end
+
+      def method_missing(method, *args, &block)
```

Comment:
It's fine as-is, I guess method missing will only be called privately anyway, just never thought of it before :smile: